### PR TITLE
Update csp-adapter version table (rancher release 2.10)

### PR DESCRIPTION
**Pull request by gbuenodevsuse**
## Description
This PR updates rancher/docs to the latest csp-adapter version available

### CODE REVIEW

File: versioned_docs/version-2.10/integrations-in-rancher/cloud-marketplace/aws-cloud-marketplace/install-adapter.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ In order to deploy and run the adapter successfully, you need to ensure its vers
 
 | Rancher Version | Adapter Version  |
 |-----------------|------------------|
+| v2.10.0         | v105.0.0+up5.0.1 |
```
Comment (LucasSaintarbor) [inline versioned_docs/version-2.10/integrations-in-rancher/cloud-marketplace/aws-cloud-marketplace/install-adapter.md:22]: @gbuenodevsuse May you remove the v2.9 entries? Thanks

**Review by LucasSaintarbor — CHANGES_REQUESTED** [review]
@gbuenodevsuse Thanks for adding the Adapter Version for v2.10. Requested changes to remove v2.9 entries.

**Comment by gbuenodevsuse** [comment]
@LucasSaintarbor, entries removed as requested!

**Comment by LucasSaintarbor** [comment]
> @LucasSaintarbor, entries removed as requested!

@gbuenodevsuse Thank you!

**Review by LucasSaintarbor — APPROVED** [review]

**Review by sunilarjun — APPROVED** [review]

**Pull request merged**